app/waf/src/sequence.ts

```diff
@@ -41,7 +49,7 @@ export class MySequence implements SequenceHandler {
       this.reject(context, err);
     }
 
-    // await this.logResponse(logUuid, context, result);
+    await this.logResponse(logUuid, context, result);
```

Comment:
Did we fix the bug of response log?

Reply:
Not yet. commented on it and Add a TODO.